content/visual-diffs/configure-visual-diffs.md

```diff
@@ -1,40 +1,51 @@
 ---
 title: "Configure Visual Diffs"
 date: 2019-09-26T15:29:49-04:00
+lastmod: 2020-09-24T10:00:00-04:00
 weight: 1
 ---
 
-To configure which pages have Visual Diffs generated, specify the _relative URLs_ of the pages in a `visualdiffs` key in
-the [service definition](/setting-up-services/). Each URL can be either a string, or a map that overrides the default
-screenshot options. For more info on exactly what you can use in this key, check out:
-[Tugboat Configuration -> visualdiffs](/reference/tugboat-configuration/#visualdiffs).
+## Prerequisite: Use Base Previews
+
+In order to configure Tugboat's visual diffs, you must be using at least one
+[Base Preview](/building-a-preview/work-with-base-previews/). For more info, see:
+[How Visual Diffs work](../using-visual-diffs/).
+
+### How to configure visual diffs
+
+- [Simple visualdiff configuration](#simple-visualdiff-configuration)
+- [Group visualdiffs via service alias](#group-visualdiffs-via-service-alias)
+- [Use advanced visualdiff configuration options](#use-advanced-visualdiff-configuration-options)
+- [Specify screenshot and visualdiff settings explicitly](#specify-screenshot-and-visualdiff-settings-explictly)
+- [Turn off visual diffs](#turn-off-visual-diffs)
+
+## Simple visualdiff configuration
+
+To configure which pages have Visual Diffs generated, specify the _relative URLs_ of the pages in a `urls` key in the
+[service definition](/setting-up-services/). In this format, each URL can be a string, and you can list multiple urls.
 
 ```yaml
 services:
   apache:
-    visualdiffs:
-      # Create a visualdiff of the home page using the default options
+    urls:
+      # Create visual diffs of the these URLs using the default options
       - /
-
-      # Create a visualdiff of /blog, but override the default timeout option
-      - url: /blog
-        timeout: 10
-
-      # Create a visualdiff of /about, but override the default waitUntil option
-      - url: /about
-        waitUntil: domcontentloaded
+      - /blog
+      - /about
 ```
 
-The URL list can also be grouped by [service alias](/reference/tugboat-configuration/#aliases), which is convenient when
-aliases have different URL structures
+## Group visualdiffs via service alias
+
+You can also group URLs by [service alias](/reference/tugboat-configuration/#aliases), which is convenient when aliases
+have different URL structures.
 
 ```yaml
 services:
   apache:
     aliases:
       - foo
 
-    visualdiffs:
+    urls:
       # Create visualdiffs for the default alias
       :default:
         - /
```

Comment:
I can't comment on the right line because it's not part of the diff, but on line 59, the comment about ignoring the URL is inaccurate. Same as the note I added in configure-lighthouse-reports.md

Reply:
Ahh, ok, got it! Ignore my comment above. I'll delete it from both places.